docs/learn/learn-proxies.md

```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
-- **Step 4**: Call extrinsic `proxy.killAnonymous` using the selected account ANON PROXY and the following parameters:
+Anon accounts cannot sign anything because they do not have private keys. As a consequence, Anon cannot act as proxies (which is admittedly confusing given their original name), and cannot be included in multi-sig. An Anon account acting as a proxy will not be able to sign anything on behalf of the primary account, and an Anon within a multi-sig will be able neither to initiate nor to approve any call.
 
-  - Spawner: (original account)
-  - Proxy type (kind of proxy)
-  - Index 0 (almost always, but can be seen in creating extrinsic)
-  - Block number x
-  - Extrinsic index y
+:::danger 
 
-  ![call extrinsic](../assets/kill-proxy-3.png)
+Once you remove the relationship with `Any` proxy, the Anon will be inaccessible. Also, Anon accounts cannot sign for anything. For example, do not include Anon in a multi-sig account.
 
-- **Step 5**: Submit and sign extrinsic
+:::
 
-  ![sign extrinsic](../assets/kill-proxy-3.png)
+:::caution Removing anonymous proxies
 
-## How to view your Proxies
+The procedure for removing an Anon is different from the one used to remove proxies. Visit the section "Removing an Anonymous Proxy" on [this support article](https://support.polkadot.network/support/solutions/articles/65000182196), or watch [this technical explainer video](https://www.youtube.com/watch?v=T443RcCYP24).
 
-To view your proxy, head over to the Chain State (underneath "Developer") page on
-[Polkadot-JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/chainstate). If
-you've created your proxy on a Kusama account, it is required to change your network accordingly
-using the top left navigation button. On this page, the proxy pallet should be selected, returning
-the announcements and proxies functions. The proxies function will allow you to see your created
-proxies for either one account or for all accounts (using the toggle will enable this). Proxy
-announcements are what time lock proxies do to announce they are going to conduct an action.
+:::
 
-![view proxies](../assets/polkadot_view_proxies.png)
+## Why Anonymous Proxies?
 
-## Putting It All Together
+Despite their complexity and associated dangers, Anon accounts have important benefits. We discuss these benefits below.
 
-If the idea of proxy types and their application seems abstract, it is. Here is an example of how
-you might use these accounts. Imagine you have one account as your primary token-holding account,
-and don't want to access it very often, but you do want to participate in governance and staking.
-You could set Governance and Staking proxies.
+### Enhanced Security
+Anon accounts cannot be stolen because they do not have private keys. The only accounts that have full access to the Anon are `Any` proxies. Security can be further increased if the `Any` proxy is a multi-signature account.
```

Comment:
```suggestion
Anon accounts cannot be stolen because they do not have private keys. The only accounts that have full access to the `Anon` are `Any` proxies. Security can be further increased if the `Any` proxy is a multi-signature account.
```